scripts/travis/build.sh

```diff
@@ -34,16 +34,17 @@ SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
 OS=$("${SCRIPTPATH}/../ostype.sh")
 ARCH=$("${SCRIPTPATH}/../archtype.sh")
-# Use go version specified by get_golang_version.sh
-if ! GOLANG_VERSION=$("${SCRIPTPATH}/../get_golang_version.sh")
+
+if ! ./scripts/check_golang_version.sh
 then
-    echo "${GOLANG_VERSION}"
     exit 1
 fi
+# Get the go build version.
+GOLANG_VERSION=$(./scripts/get_golang_version.sh)
 
 curl -sL -o ~/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
 chmod +x ~/gimme
-eval $(~/gimme "${GOLANG_VERSION}")
+eval "$(~/gimme "${GOLANG_VERSION}")"
```

Comment:
This satisfies shellchecker.

Reply:
Are you able to use double quotes again like that inside of `$()`?

Reply:
Yes.  The outer quotes would be expanding anything returned by the command, while the inner quotes are still interpolating the variable.